src/main/java/org/reaktivity/nukleus/tls/internal/stream/ServerStreamFactory.java

```diff
@@ -766,14 +778,38 @@ private void handleReset(
         {
             try
             {
-                doReset(networkThrottle, networkId);
                 doCloseInbound(tlsEngine);
             }
             catch (SSLException ex)
             {
-                LangUtil.rethrowUnchecked(ex);
+                // Ignore and clean up
+            }
+            finally
+            {
+                doResetApplicationReplyStream();
+            }
+        }
+
+        private void doAbortApplicationStream()
+        {
+            doAbort(applicationTarget, applicationId);
+        }
+
+        private void doResetApplicationReplyStream()
+        {
+            correlations.remove(appCorrelationId);
+
+            if (applicationReplyCleaner != null)
+            {
+                applicationReplyCleaner.run();
             }
         }
+
+        private void setApplicationReplyCleaner(
+            Runnable applicationReplyCleaner)
+        {
+            this.applicationReplyCleaner = applicationReplyCleaner;
+        }
```

Comment:
Rename as `setNetworkReplyDoneHandler` method and `networkReplyDoneHandler` field and parameter.

Reply:
done